docker-compose.yml

```diff
@@ -122,6 +122,7 @@ services:
       # Registry
       - "80:80"
       - "443:443"
+      - "7000:7000" # epp_proxy middleware
```

Comment:
Why are you exposing 7000 here if the app is ran inside epp container ?

Reply:
Rudiment from testing when I was uncertain how to compose the whole environment. Thanks for pointing it out!